de.tubs.variantsync.core/src/de/tubs/variantsync/core/view/featurecontext/FeatureContextWizardPage.java

```diff
@@ -147,12 +147,14 @@ public void handleEvent(Event event) {
 				final TableItem[] selectedItem = tabFeatures.getTable().getSelection();
 				if (selectedItem.length > 0) {
 					String featureName = selectedItem[0].getText();
+					// if the featureName is of the following structure:
+					// arbitrary amount of chars until the first occurence of 1 or more consecutive spaces followed by an arbitrary amount of chars
```

Comment:
If not relevant for now you can leave as is.

Reply:
this if-statement simply means, that when the feature name contains one or more spaces, it is surrounded by double quotes.
I think this is done because the space character is needed to put features together by operators like `and` or `or`

Reply:
Ok could you add that as a comment?

Reply:
already done, but not pushed yet

Reply:
this might also be a relict of previous versions because it is not even possible to add new features to the feature model which contain a space character in the middle. so this check seems unnecessary?!?!

Reply:
Yes spaces should be forbidden and I experienced issues with this in FeatureIDE, too.